db/connect.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose')
+
+
+
```

Comment:
You can enter you connection string here for the start, in the future you well lean about saving secrets & special configurations